frontend/src/components/common/render-html/render-html.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'common/types/types';
+import { sanitizeHTML } from 'helpers/helpers';
+
+type Props = {
+  html: string;
+  className?: string;
+};
+
+const RenderHTML: FC<Props> = ({ html, className }) => {
```

Comment:
```suggestion
const Content: FC<Props> = ({ html, className }) => {
```
don't forget about file name